google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -215,20 +215,19 @@ void WaitForConsistencyCheck(google::cloud::bigtable::TableAdmin admin,
 
 void CheckConsistency(google::cloud::bigtable::TableAdmin admin, int argc,
                       char* argv[]) {
-  if (argc != 3) {
-    throw Usage{
-        "check-consistency: <project-id> <instance-id> <table-id> "
-        "<consistency_token>"};
+  if (argc != 2) {
+    throw Usage{"check-consistency: <project-id> <instance-id> <table-id>"};
   }
   std::string const table_id_param = ConsumeArg(argc, argv);
-  std::string const consistency_token_param = ConsumeArg(argc, argv);
```

Comment:
Why did we drop this?

Reply:
My thought was as below 
 
1. if we do no want to add in ci build, then we can keep this user token as a function parameter
2. if we want to add in ci build, then seems no use of user token as a function parameter, and use GenerateConsistencyToken inside the example only. 
3. if we want to add in ci build, and also keep user token as a function parameter, then need to check more, how to do that, or if you have any suggestion.

Reply:
I think we can do option 3 without much effort:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/ad0601fd8e3b8cee786fb5bb472f4479322740a3/google/cloud/storage/examples/run_examples_utils.sh#L231